tests/test_client.py

```diff
@@ -937,6 +943,25 @@ def test_get_order_decisions(self):
             assert(response.body['decisions']['payment_abuse']['decision']['id'] == 'decision7')
             assert(response.body['decisions']['promotion_abuse']['decision']['id'] == 'good_order')
 
+    def test_get_session_decisions(self):
+        mock_response = mock.Mock()
+        mock_response.content = '{"decisions":{"payment_abuse":{"decision":{"id":"user_decision"},"time":1468707128659,"webhook_succeeded":false}}}'
```

Comment:
More realistic example will be `account_takeover` instead of `payment_abuse`. It will be also better if we have `session_decision` as an ID, as we're adding a function to query session decisions.

As for the examples in the real world, you can actually query it via api3 client: https://paper.dropbox.com/doc/API3-Client-ueDRYD8Bza9k6FEGugHeo. We can sync up on how to use this tool later.